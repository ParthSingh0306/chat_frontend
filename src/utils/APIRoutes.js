export const host = "https://chat-backend-hb9o.onrender.com";
export const loginRoute = `${host}/api/v1/users/login`;
export const registerRoute = `${host}/api/v1/users/register`;
export const logoutRoute = `${host}/api/v1/users/logout`;
export const allUsersRoute = `${host}/api/v1/users/allusers`;
export const setAvatarRoute = `${host}/api/v1/users/setavatar`;
export const sendMessageRoute = `${host}/api/v1/messages/addmsg`;
export const recieveMessageRoute = `${host}/api/v1/messages/getmsg`;
