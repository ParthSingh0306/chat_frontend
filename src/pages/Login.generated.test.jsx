import renderer from 'react-test-renderer';
import React, { useState, useEffect } from "react";
import axios from "axios";
import { useNavigate, Link } from "react-router-dom";
import Logo from "../assets/logo.svg";
import { loginRoute } from "../utils/APIRoutes";
import toast from "react-hot-toast";
import Login from "./Login";

jest.mock("axios");
jest.mock("react-router-dom");
jest.mock("../assets/logo.svg");
jest.mock("../utils/APIRoutes");
jest.mock("react-hot-toast");

const renderTree = tree => renderer.create(tree);
describe('<Login>', () => {
  it('should render component', () => {
    expect(renderTree(<Login 
    />).toJSON()).toMatchSnapshot();
  });
  
});