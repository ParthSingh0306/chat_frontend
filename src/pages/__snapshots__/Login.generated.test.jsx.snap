// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login> should render component 1`] = `
<div
  className="sc-beySPh jnYhCz"
>
  <form
    action=""
    onSubmit={[Function]}
  >
    <div
      className="brand"
    >
      <img
        alt="logo"
        src="logo.svg"
      />
      <h1>
        snappy
      </h1>
    </div>
    <input
      min="3"
      name="username"
      onChange={[Function]}
      placeholder="Username"
      type="text"
    />
    <input
      name="password"
      onChange={[Function]}
      placeholder="Password"
      type="password"
    />
    <button
      type="submit"
    >
      Log In
    </button>
    <span>
      Don't have an account ? 
    </span>
  </form>
</div>
`;
